jwst-masterclass/start_jd.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -eux
+
+ln -s "$CONDA_DIR/vnc/Desktop"
```

Comment:
Consider adding the symlink to home_template so we can use the base notebook CMD

Reply:
This file isn't actually needed.